src/test/HomePage.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import { BrowserRouter as Router } from 'react-router-dom'
+import HandleError from 'utils/HandleError'
+import Content from 'Components/Content'
+import HomePage from 'Pages/HomePage'
+
+describe('Fetch and render from the API', () => {
+    const HomePageStructure =
+    <Router>
+        <HandleError message={"We're sorry, something went wrong"}>
+            <Content>
+                <HomePage>
+                </HomePage>
+            </Content>
+        </HandleError>
+    </Router>
+
+    test('Featured Banners are fetched and renders correctly', async () => {
```

Comment:
Try checking if the loader appears before the banners are shown, this will give you the correct expected behavior.